inst/private/python_ipc_system.m

```diff
@@ -113,12 +113,21 @@
         disp (['This session will use the temp file "' tmpfilename '"'])
       end
     end
-    fd = fopen (tmpfilename, 'w');
+    [fd, msg] = fopen (tmpfilename, 'w');
+    if fd < 0
+      error (msg)
+    end
```

Comment:
Should we prepend `system ipc:` to the error message and show the temp file name as well?

```
      error ('system ipc: failed to create temporary file %s: %s',
             tmpfilename, msg);
```